### ISSUE TITLE
[Enhancement] Improve UI

### ISSUE THREAD

**Issue by Cyrexxis** [Cyrexxis/RokTracker#114]
**Is your feature request related to a problem? Please describe.**
The UI is very slow and there are many different exe files.

**Describe the solution you'd like**
Use a webview based approach (something like pywebview in combination with vue). In combination to that also use the same exe for all scanners and use a navigatoin bar to switch the modes.

**Describe alternatives you've considered**
Using native libraries like pyqt or wxPython, but these seem very complicated. Another opion would be to try normal tkinter (instead of customtkinter) but that also has the reputation of being slow.

**Comment by AbdUlrahman097** [comment]
@Cyrexxis no matter what i do the webview window displaying my Vue.js doesn't work at all so i used the alternatives as you said using pyqt5 on seed scanner and it worked very good

**Comment by Cyrexxis** [comment]
I am currently testing the webview approach locally and for me, it works well. I want to implement at least the kingdom scanner before I commit it to this repository.
For me, webview is easier because I have more experience with it. Also, I do not understand the licensing of QT, in my opinion it is very strange.

**Comment by AbdUlrahman097** [comment]
![Image](https://github.com/user-attachments/assets/68d630c7-28a7-4da9-b5bc-244ef7450eda)
This is what I was able to achieve with a little help from an AI but no progress anyway, Thanks for the good and difficult ideas and for your effort too

**Comment by Cyrexxis** [comment]
Just a update with a Screenshot here. I use Vue + Quasar for the webapp and run that with the pywebview package. Currently there is no interaction with the python backend yet and some important elements of the UI are missing.

![Current Status](https://github.com/user-attachments/assets/04121d8d-4a13-40e6-b607-11aa7dfa411c)

**Comment by AbdUlrahman097** [comment]
@Cyrexxis Just wanted to commend you on the seamless performance of making all scanners work in the New UI, excellent execution!

On a related note, I’ve been experimenting with a modified version of the UI in my fork, migrating from CustomTkinter to PyQt6 for improved scalability and performance. I’ve also incorporated a few new features to enhance functionality and user experience. I’d like to share this with you for feedback, if you’re interested!

![Image](https://github.com/user-attachments/assets/1411fea5-e068-4e2e-8918-f3aa22eef955)

![Image](https://github.com/user-attachments/assets/8b208411-4584-4376-9b95-d4ac3e7fa19d)

**Comment by MakaveIli** [comment]
Looks great, do you share this

**Comment by AbdUlrahman097** [comment]
yes this is in my fork here https://github.com/AbdUlrahman097/New-RokTracker
but i didn't publish the analytics feature yet
